src/js/tabs/motors.js

```diff
@@ -222,6 +226,15 @@ TABS.motors.initialize = function (callback) {
         }
 
         $('.mixerPreview img').attr('src', './resources/motor_order/' + mixerList[mixer - 1].image + reverse + '.svg');
+
+        const motorOutputReorderConfig = new MotorOutputReorderConfig(100);
+        const domMotorOutputReorderDialogOpen = $('#motorOutputReorderDialogOpen');
+
+        if (mixerList[mixer - 1].name in motorOutputReorderConfig && MOTOR_OUTPUT_ORDER.length > 0) {
+            domMotorOutputReorderDialogOpen.show();
+        } else {
+            domMotorOutputReorderDialogOpen.hide();
+        }
```

Comment:
Only as a comment, not need to change, we use both ways in the code...

For this kind of open/hide elements usually is cleaner and more visible something like:
```javascript
const isReorderMotorsAvailable = mixerList[mixer - 1].name in motorOutputReorderConfig && MOTOR_OUTPUT_ORDER.length > 0;
domMotorOutputReorderDialogOpen.toggle(isReorderMotorsAvailable);
```